ibm_db2/README.md

```diff
@@ -168,6 +168,21 @@ See [service_checks.json][11] for a list of service checks provided by this inte
 
 ## Troubleshooting
 
+### CLI Driver SQL1531N error
+
+If you encounter an issue that produces error logs like the following:
+
+```
+2023-08-10 23:34:47 UTC | CORE | ERROR | (pkg/collector/python/datadog_agent.go:129 in LogMessage) | ibm_db2:c051131490335a94 | (ibm_db2.py:563) | Unable to connect to database `datadog` as user `db2inst1`: [IBM][CLI Driver] SQL1531N  The connection failed because the name specified with the DSN connection string keyword could not be found in either the db2dsdriver.cfg configuration file or the db2cli.ini configuration file.  Data source name specified in the connection string: "DATADOG". SQLCODE=-1531
+```
+
+Then it's most likely caused by one of the following scenarios:
+- The configuration (conf.yaml) is missing a host and port configuration
```

Comment:
I see that host is optional but it does not default to localhost. What happens when no host is provided?